src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -46,6 +46,8 @@ internal static class Constants
         public const string DefaultManagementURL = "https://management.azure.com/";
         public const string AzureManagementAccessToken = "AZURE_MANAGEMENT_ACCESS_TOKEN";
         public const string ExtensionBundleConfigPropertyName = "extensionBundle";
+        public const string OSXCoreToolsTempDirectoryName = ".azure-functions-core-tools";
+        public const string OSXRootPath = "~/";
```

Comment:
Can probably make this `~`. But either way should work (I think?).

Reply:
Yeah i checked both notation as same. Keep this as is.